Dockerfile

```diff
@@ -1,18 +1,18 @@
-# Use Linux-Alpine image
```

Comment:
Why remove this comment? Is it causing an issue or outdated info?

Reply:
It wasn't causing an issue nor was it outdated, but we can see we're using Alpine Linux due to it being called out directly in the line below.  The `-alpine` tag is a very common way to dictate that we're using Alpine Linux.